lib/livebook_web/live/session_live/app_teams_live.ex

```diff
@@ -343,7 +343,7 @@ defmodule LivebookWeb.SessionLive.AppTeamsLive do
     with {:ok, app_deployment} <- pack_app(socket),
          :ok <- deploy_app(socket, app_deployment) do
       message =
-        "App deployment for #{app_deployment.slug} with title #{app_deployment.title} created successfully."
+        "App deployment for #{app_deployment.slug} with name #{app_deployment.title} created successfully."
```

Comment:
Here is another option:

```suggestion
        "Notebook #{app_deployment.title} was deployed with success to #{app_deployment.slug}."
```

But we don't have to even include the title.

Reply:
And perhaps we change the modal title to "Deploy with Livebook Teams" as well. I don't think we should necessarily say the words "app deployment" to users. Thoughts?

Reply:
Regarding the message, I'm ok with just"

  "App deployment created successfully"

Without mentioning the name/title or the slug. Once we implement the "list of apps" feature in Teams, we'll probably have to add a link to that page in that message or page.

Reply:
I understand your point of view and I am fine with whatever option you pick. :)